docs/admin-applic-config.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
I would remove this file, the documentation does not provide enough guidance beyond what our interface provides already.